plugins/tracklabels/init.js

```diff
@@ -123,7 +123,7 @@ theWebUI.trackersLabelContextMenu = function(e)
 	return(false);
 }
 
-plugin.updateLabelsImages = function()
+plugin.updateLabelsImages = async function()
```

Comment:
We not need 'async' here. The async function declaration simply makes the function automatically return a promise.
Also you can use the await operator inside of async function. That's all.
Try to check something like this:
```
async function f(){
    console.log(2);
}

console.log(1);
f();
console.log(3);
```
You will have "1 2 3", not "1 3 2".

Reply:
Okay, I have actually made this task async using `setTimeout`. The function is no longer returning a promise to `theWebUI.updateLabels`. I agree this was undesirable behavior. It's also a micro-optimization to do this because we're mitigating (in some cases removing) a forced reflow during `theWebUI.addTorrents`.

The real reason though is because other tasks like `cpuload` and `diskspace` were waiting on `tracklabels/action.php`. In some cases they were delayed by 200ms during startup. These tasks are higher priority tasks because they require a network transfer to complete. The labels can be loaded from cache anytime, but these need to start right away.

Reply:
> Okay, I have actually made this task async using setTimeout

This task *already* is asynchronous. Function ```updateLabelsImages``` simple modify a set of ```src``` attributes. It does not wait the result of loading of corresponding images.
Your changes in this fragment is useless, IMHO.

Reply:
I am using the performance profiler on Google Chrome. A forced reflow was eliminated, reducing the time it takes to load labels from 10ms to 3ms. But the real benefit is `cpuload/action.php` and `diskspace/action.php` were initiated before `tracklabels/action.php`. This made the cpu load and disk space icons load 150ms faster. They must retrieve the percentage used from the web server using a network request, before the icons can be loaded.

Reply:
> A forced reflow was eliminated, reducing the time it takes to load labels from 10ms to 3ms

Sorry, but I think this is a simple measurement error. 

> But the real benefit is cpuload/action.php and diskspace/action.php were initiated before tracklabels/action.php

For example, on the my installation (master branch) they *already* initiated before, and your changes haven't relation to this.
First request of cpuload:
![2022-01-13_21h53_17](https://user-images.githubusercontent.com/1230409/149391968-491a9868-95f1-4a3f-b619-0907a76bc076.png)

First request of diskspace:
![2022-01-13_21h59_35](https://user-images.githubusercontent.com/1230409/149392457-9be5d4e2-a0d4-41a7-8151-46868f4223a1.png)

First request of tracklabels:
![2022-01-13_21h53_44](https://user-images.githubusercontent.com/1230409/149391977-a7e61c7b-ea34-48ba-a14c-c43ad459f9a0.png)

Reply:
That's not how it working on my waterfall page. But I would like this pull request merged, so I reverted this change. I'll take what I can get. It's better to have two changes incorporated into master than none.